src/azure-cli/azure/cli/command_modules/acr/token.py

```diff
@@ -39,6 +51,51 @@ def acr_token_create(cmd,
         )
     )
 
+    if no_passwords:
+        return poller
+
+    token = LongRunningOperation(cmd.cli_ctx)(poller)
+    _create_default_passwords(cmd, resource_group_name, registry_name, token, logger)
+    return token
+
+
+def _create_default_scope_map(cmd, resource_group_name, registry_name, token_name, repositories, logger):
+    from ._client_factory import cf_acr_scope_maps
+    scope_map_name = DEF_SCOPE_MAP_NAME_TEMPLATE.format(token_name)
+    scope_map_client = cf_acr_scope_maps(cmd.cli_ctx)
+    actions = parse_actions_from_repositories(repositories)
+    try:
+        existing_scope_map = scope_map_client.get(resource_group_name, registry_name, scope_map_name)
+        # for command idempotency, if the actions are the same, we accept it
+        if sorted(existing_scope_map.actions) == sorted(actions):
+            return existing_scope_map.id
+        raise CLIError('The default scope map was already configured with different respository permissions. Please'
+                       ' use "az acr scope-map update -r {} -n {} --add <REPO> --remove <REPO>" to update'.format(
+                           registry_name, scope_map_name))
+    except CloudError:
+        pass
+    logger.warning('Creating a scope map of "%s" for provided respository permissions', scope_map_name)
+    poller = scope_map_client.create(resource_group_name, registry_name, scope_map_name,
+                                     actions, "Token {}'s scope map".format(token_name))
```

Comment:
Not sure if the scope-map `description` is generic enough since one scope-map can be tied to multiple tokens.
How about: `"Created by token: {}".format(token_name)`